src/content/11/en/part11a.md

```diff
@@ -15,11 +15,33 @@ If we're both working on changes and we haven't decided on a way to deploy to pr
+In this module we'll be using some terms you may not be familiar with or you may not have a good understanding of. We'll discuss some of these terms here. Even if you are familiar with the terms, give this section a read so when we use the terms in this module, we're on the same page.
+
+#### Branches
+Git allows multiple copies, streams, or versions of the code to co-exist without overwriting eachother. When you first create a repository, you will be looking at the main branch (usually in git, we call this `master` or `main`, but that does vary in older projects). This is fine if there's only one developer for a project and that developer only works on one feature at a time.
+
+Branches are useful when this environment becomes more complex. In this context each developer can have one or more branches. Each branch is effectively a copy of the main branch with some changes that make it diverge from master. Once the feature or change in the branch is ready it can be merged back into the main branch, effectively making that feature or change part of the main software. In this way each developer can work on their own set of changes and not affect any other developer until the changes are ready. 
+
+But once one developer has merged their changes into the main branch, what happens to the other developers' branches? They are now diverging from an older copy of the main branch. How will the developer on the later branch know if their changes are compatible with the current state of the main branch? That is one of the fundamental questions we will be trying to answer in this module.
+
+#### Build
+The term 'build' has different meanings in different languages. In some languages, there is no need for a build step. 
+
+In general when we talk about building we mean preparing software to run on the platform where it's intended to run. This might mean, for example, that if you've written your application in TypeScript, and you intend to run it on Node, then the build step might be transcoding the TS into JS.
+
+This step is much more complicated (and required) in compiled languages where the code needs to be compiled into an executable.
+
+#### Deploy
+Deployment refers to putting the software where it needs to be for the end user to use it. In the case of libraries, this may simply mean pushing an npm package to a package archive (such as npmjs.com) where other users can find it and include it in their software. 
+
+Deploying a service (such as a web app) may be a little more complicated. In this module we'll cover a simple deployment to a web based hosting service where users can use the software simply by going to the URL. We won't cover it in this module but it's important to know that deployments can be significantly more complex especially if we add requirements such as "the software must be available at all times during the deployment" (zero downtime deployments) or if we have to take things like database migrations into account.
```

Comment:
> Deploying a service (such as a web app) may be a little more complicated. In this module we'll cover a simple deployment to a web based hosting service where users can use the software simply by going to the URL. We won't cover it in this module but it's important...

This part took me a few read-throughs. We should cover basic deployment of a service or such. We don't need to go all-in on it by taking in also things like zero downtime deployments which could be considered a more advanced/refined deployment procedure.